Source/WebCore/platform/Scrollbar.cpp

```diff
@@ -539,4 +539,13 @@ int Scrollbar::minimumThumbLength() const
     return m_scrollableArea.scrollbarsController().minimumThumbLength(m_orientation);
 }
 
+void Scrollbar::updateScrollbarThickness()
+{
+    if (!isCustomScrollbar()) {
+        int thickness = ScrollbarTheme::theme().scrollbarThickness(widthStyle());
+        setFrameRect(IntRect(0, 0, thickness, thickness));
```

Comment:
This seems wrong (since the scrollbar may have been resized since creation). You can set both height and width to the thickness. If this is a real bug, it should be covered by a test.

Reply:
Not sure what you mean here. This is following the existing code in `Scrollbar`. This comment seems relevant though: 
>   // FIXME: This is ugly and would not be necessary if we fix cross-platform code to actually query for
>     // scrollbar thickness and use it when sizing scrollbars (rather than leaving one dimension of the scrollbar
>     // alone when sizing).

Reply:
It would be clearer to only touch the thickness axis.